Release Candidate v0.0.2 (#71)
pysat/pysatNASA: docs/examples/ex_init.rst

Loading ICON IVM data
=====================

pysatNASA uses `pysat <https://github.com/pysat/pysat>`_ to load
space science instrument data.  As specified in the
`pysat tutorial <https://pysat.readthedocs.io/en/latest/tutorial.html>`_,
data may be loaded using the following commands.  Data from the Ion Velocity

> Meter on board the Ionospheric CONnection Explorer (ICON) is used as an example.

Reviewer: ```suggestion
Meter on board the Ionospheric CONnection Explorer `(ICON) <https://www.nasa.gov/icon>`_ used as an example.
```

::

   import datetime as dt
   import pysat
   import pysatNASA as py_nasa

> 

Reviewer: How about we register pysatNASA with pysat?
```suggestion
pysat.utils.registry.register_by_module(py_nasa.instruments)

```

   old_time = dt.datetime(2020, 1, 1)

>    ivm = pysat.Instrument(inst_module=py_nasa.instruments.icon_ivm,

Reviewer: ```suggestion
   ivm = pysat.Instrument(platform='icon', name='ivm',
```
